src/modules/cardTypes/cardTypes.service.ts

```diff
@@ -96,66 +99,71 @@ export class CardTypesService {
 
   update = async (updateCardTypesDTO: UpdateCardTypesDTO) => {
     try {
-      const foundCardTpyes = await this.cardTypesRepository.findOneBy({
+      const foundCardTypes = await this.cardTypesRepository.findOneBy({
```

Comment:
it makes more sense to me to rename to "currentCardType" because cardTypes is for more than 1, and we're just fetching one card

Reply:
Ready